src/tree.ts

```diff
@@ -263,17 +246,26 @@ const makeSourcesTree = (handle: FileHandle, options: any): Source[] => {
   };
   // Handle three cases -- sizes + type, just sizes, just type
   if (!options.sizes && options.formats) {
-    return R.reject(R.isNil, R.map((f: string) => makeSource(null, null, f), options.formats));
+    const sources = options.formats.map((format: string) => {
```

Comment:
You dont need to use return, just make from it one liner, it will be much cleaner